service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -642,4 +643,6 @@ public boolean hasFutureHearing(List<Element<HearingBooking>> hearingBookings) {
             .filter(hearingBooking -> hearingBooking.getStartDate().isAfter(currentCmoStartDate))
             .min(comparing(HearingBooking::getStartDate));
     }
+
+    private String sendToCtsc;
```

Comment:
nice. I created a sub-task for this and other scenarios like this - 

https://tools.hmcts.net/jira/browse/FPLA-2148